clients/vscode/src/code-action/QuickFix.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  CancellationToken,
+  CodeAction,
+  CodeActionContext,
+  CodeActionKind,
+  CodeActionProvider as CodeActionProviderInterface,
+  Range,
+  Selection,
+  TextDocument,
+} from "vscode";
+import { ContextVariables } from "../ContextVariables";
+import { getLogger } from "../logger";
+export class QuickFixCodeActionProvider implements CodeActionProviderInterface {
+  constructor(private readonly contextVariables: ContextVariables) {}
+
+  provideCodeActions(
+    _document: TextDocument,
+    _range: Range | Selection,
+    context: CodeActionContext,
+    token: CancellationToken,
+  ): CodeAction[] | undefined {
+    if (token.isCancellationRequested) {
+      return;
+    }
+
+    if (!this.contextVariables.chatEnabled) {
+      return;
+    }
+    if (context.diagnostics.length === 0) {
+      return [];
+    }
+
+    //TODO: Get all diagnostics from the context and substring with max command chars
+    const lspErrors = context.diagnostics[0];
```

Comment:
```suggestion
    const lspErrors = context.diagnostics;
```

Reply:
I tried passing all errors before, but unfortunately, our backend did check max command chars. I did try to minimize the length of the commands. Previously, I was sending out all the error messages, but unfortunately, our default max command char is only 200. Multiple error messages can easily exceed this limit. Is it okay for me to change the default max char?

Reply:
For example: 
![image](https://github.com/user-attachments/assets/4e0bda60-50a4-4ce5-9d75-7427401970e9)

Therefore, I change it to 600